### PR TITLE
GrafanaUI: Fixes Chrome issue for various query fields

### LINKED ISSUE #54535: QueryFields/Input fields are broken in the recent Chrome browser
Initially reported [here](https://raintank-corp.slack.com/archives/C13QEUGLB/p1661934429060559).

It seems that there are some inputs, or combination of inputs, that potentially cause cursor jumps/focus loss on the code editor.

Some examples both in Firefox and Chrome:

https://user-images.githubusercontent.com/1069378/187705345-3cc11c1f-12e1-407c-886b-74b59d81fbe5.mov

https://user-images.githubusercontent.com/1069378/187705346-e1b2a864-8046-4e27-b6cb-182e4f43703c.mov

### PR DESCRIPTION
**Pull request by kaydelaney**
**What this PR does / why we need it**:
Bumps slate (the component used by the QueryField) and changes the version from the old fork I made to the upstream version. This fixes the issue with chrome browsers

Fixes #54535

### CODE REVIEW

File: public/app/features/plugins/plugin_loader.ts [hunk 1]
```diff
@@ -85,7 +85,7 @@ exposeToPlugin('react-router-dom', reactRouter);
 // Experimental modules
 exposeToPlugin('prismjs', prismjs);
 exposeToPlugin('slate', slate);
-exposeToPlugin('@grafana/slate-react', slateReact);
+exposeToPlugin('slate-reacvt', slateReact);
```
Comment (ashharrison90) [inline public/app/features/plugins/plugin_loader.ts:88]: typo here 😱

File: .betterer.results [hunk 2]
```diff
@@ -1925,41 +1924,18 @@ exports[`better eslint`] = {
...
-      [0, 0, 0, "Do not use any type assertions.", "3"],
-      [0, 0, 0, "Unexpected any. Specify a different type.", "4"],
-      [0, 0, 0, "Do not use any type assertions.", "5"]
-    ],
-    "packages/grafana-ui/src/slate-plugins/indentation.ts:5381": [
-      [0, 0, 0, "Do not use any type assertions.", "0"]
-    ],
-    "packages/grafana-ui/src/slate-plugins/newline.ts:5381": [
-      [0, 0, 0, "Do not use any type assertions.", "0"]
-    ],
```
Comment (ashharrison90) [inline .betterer.results:1952]: ohhhhhh baby 😍

**Review by ashharrison90 — COMMENTED** [review]
well this looks 😍😍😍😍😍😍

think the slate deps can now be removed from the ignore list in `renovate.json5`

will pull down and play around in the UI when i get a chance

File: public/app/features/plugins/plugin_loader.ts (hunk 1 above)
Comment (kaydelaney) [reply public/app/features/plugins/plugin_loader.ts:88]: Oops! Thanks for catching that

File: package.json [hunk 3]
```diff
@@ -404,13 +404,14 @@
...
-    "@types/slate": "0.47.2",
+    "@types/slate": "0.47.9",
     "@microsoft/api-extractor-model": "7.23.3",
     "@rushstack/node-core-library": "3.49.0",
     "@rushstack/rig-package": "0.3.13",
     "@rushstack/ts-command-line": "4.12.1",
     "@storybook/react/webpack": "5.74.0",
-    "node-fetch": "2.6.7"
+    "node-fetch": "2.6.7",
+    "slate-dev-environment@^0.2.2": "patch:slate-dev-environment@npm:0.2.5#.yarn/patches/slate-dev-environment-npm-0.2.5-9aeb7da7b5.patch"
```
Comment (svennergr) [inline package.json:414]: This is working for Grafana now, but should we incorporate that patch also/only to grafana-ui? I am not sure if there are dependents which do not those patches but would likely need them.

**Review by svennergr — APPROVED** [review]
LGTM 🚀

Great job @kaydelaney - appreciate the quick fix!

**Comment by svennergr** [comment]
Sorry, one more thing: can we potentially backport this? 9.1.x users are facing issues which will get fixed by this. (ref https://github.com/grafana/grafana/issues/54535)

**Review by ashharrison90 — APPROVED** [review]
lgtm 🙌 would be great if you could update the `renovate.json5` file as well (unless there's a reason why we shouldn't continue to update slate? 🤔 ), but if not i can do it in a follow up PR 🙂

**Comment by ethirolle** [comment]
Per this earlier comment from @svennergr:
> Sorry, one more thing: can we potentially backport this? 9.1.x users are facing issues which will get fixed by this. (ref #54535)

Is there a backport PR to get this fix into 9.1? (We need to get this fix out there asap to help our users & customers who will be affected by this difficult UI issue as soon as they upgrade Chrome/Edge to latest version...)

(And do we need backport PRs for earlier Grafana versions, like 8.5?)

Thank you!

**Comment by PrayagS** [comment]
When is this scheduled to be released? Or has a hotfix already been released?

**Comment by erkiesken** [comment]
Just noticed 9.1.4 tag is now available on Docker Hub and can verify that it fixed the issue for our company people experiencing the promql/logql editor problems in Chrome 105.x.

**Comment by WonderLand33** [comment]
> Just noticed 9.1.4 tag is now available on Docker Hub and can verify that it fixed the issue for our company people experiencing the promql/logql editor problems in Chrome 105.x.

How do I update a running docker container?  :(

**Comment by ethirolle** [comment]
@WonderLand33, I will take a shot at answering your question:

> How do I update a running docker container? :(

In general, you don't update a running Docker container (whether Grafana or any other service). Rather, you stop the container, update your deployment config (if needed) to reference the more recent version of the Grafana Docker image (like `9.1.4` or `latest`), then re-deploy the Grafana container.

**Comment by yabetsu93** [comment]
Hi @ethirolle is there any chance to apply this bug fix on 7.5.x version of grafana? since 7.5.16 release states some security fix issue would like to ask if you can do new release for 7.5.x version for this issue thanks in advance

**Comment by ethirolle** [comment]
@yabetsu93, yes! This fix has already been released in late Sept in v7.5.17: https://grafana.com/grafana/download/7.5.17. 

Normally we don't go two major versions back for fixes... BUT in this case, the decision was made to do a 7.x release that contains this fix, since there are still a lot of Grafana users on 7.x, and this Chrome issue makes the Grafana query editors pretty hard to use. The PR for this 7.5.x fix is here: https://github.com/grafana/grafana/pull/55688

**Pull request merged**